annotation/pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>com.infradna.tool</groupId>
     <artifactId>bridge-method-injector-parent</artifactId>
-    <version>1.19-SNAPSHOT</version>
+    <version>1.19</version>
```

Comment:
Not sure what is going on here.

Reply:
No such edit should ever be made as part of a functional PR. It is up to maintainers to prepare releases and any associated metadata edits. (In the case of this repository, using `maven-release-plugin`.)

Reply:
> It is up to maintainers to prepare releases and any associated metadata edits.

But they haven't done it for years... Let me revert than change.

Reply:
This project was unmaintained even before @kohsuke disappeared.